build.gradle.kts

```diff
@@ -71,7 +72,6 @@ dependencies {
     implementation("org.jetbrains.compose.material:material-icons-extended:$composeVersion")
     implementation("org.jetbrains.kotlin:kotlin-compiler-embeddable:$kotlinVersion")
     implementation("org.jetbrains.kotlin:kotlin-script-runtime:$kotlinVersion")
-    implementation("org.jetbrains.kotlin:kotlin-script-util:$kotlinVersion")
```

Comment:
This is deprecated and no longer used, I probably shouldnt have had it included in the first place